proposals/vector_buffer.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "vector_buffer",
+    "summary": "Compute a buffer around geometries",
+    "description": "Calculates a round buffer around each geometry given. Buffers can be applied to points, lines and polygons, but the results are always polygons. Multi-part types (e.g. `MultiPoint`) are also allowed.",
+    "categories": [
+        "vector"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "geometries",
+            "description": "Geometries to apply the buffer on. Vector properties are preserved for vector data cubes and all GeoJSON Features.\n\nTo maximize interoperability, a nested `GeometryCollection` should be avoided. Furthermore, a `GeometryCollection` composed of a single type of geometries should be avoided in favour of the corresponding multi-part type (e.g. `MultiPolygon`).",
+            "schema": [
+                {
+                    "type": "object",
+                    "subtype": "geojson"
+                },
+                {
+                    "type": "object",
+                    "subtype": "vector-cube"
+                }
+            ]
+        },
+        {
+            "name": "distance",
+            "description": "The radius of the buffer in the unit given in the `unit` parameter. A negative number can be provided to subtract from geometries.",
```

Comment:
> in the unit given in the `unit` parameter.

"given by the ..."?

Reply:
> A negative number can be provided to subtract from geometries.

A positive distance results in outward buffering (dilation), a negative distance results in inward buffering (erosion).

Reply:
> The radius of the buffer

Maybe better to keep using "distance" instead of introducing "radius"?

Reply:
Sounds all good. Fixed.

I also agree with distance after reading https://stackoverflow.com/a/70151787/9709414